ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -645,16 +652,17 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
     }
 
     private static func customViewLeadingOffset(isInSelectionMode: Bool,
-                                                tokenSet: TableViewCellTokenSet) -> CGFloat {
-        return tokenSet[.paddingLeading].float + selectionModeAreaWidth(isInSelectionMode: isInSelectionMode,
-                                                                        selectionImageMarginTrailing: TableViewCellTokenSet.selectionImageMarginTrailing,
-                                                                        selectionImageSize: TableViewCellTokenSet.selectionImageSize)
+                                                leadingPadding: CGFloat) -> CGFloat {
+        return leadingPadding + selectionModeAreaWidth(isInSelectionMode: isInSelectionMode,
```

Comment:
I like NOT having to pass in the entire token set for a single value. Good change.

Reply:
It was necessary before we had token constants :') Luckily no longer!